build/builder.sh

```diff
@@ -119,6 +119,9 @@ vols="${vols} --volume=${gopath0}/bin/docker_amd64:/go/bin"
 vols="${vols} --volume=${HOME}/.jspm:${container_home}/.jspm"
 vols="${vols} --volume=${HOME}/.npm:${container_home}/.npm"
 vols="${vols} --volume=${cockroach_toplevel}:/go/src/github.com/cockroachdb/cockroach"
+# Add ssh key for terraform tests.
+vols="${vols} --volume=${HOME}/.ssh/google_compute_engine.pub:${container_home}/.ssh/google_compute_engine.pub"
```

Comment:
The public key? Where is that used?

On Tue, Sep 13, 2016 at 2:24 PM, Cuong Do <notifications@github.com> wrote:

> In build/builder.sh
> <https://github.com/cockroachdb/cockroach/pull/9328#discussion_r78616263>:
>
> > @@ -119,6 +119,9 @@ vols="${vols} --volume=${gopath0}/bin/docker_amd64:/go/bin"
> >  vols="${vols} --volume=${HOME}/.jspm:${container_home}/.jspm"
> >  vols="${vols} --volume=${HOME}/.npm:${container_home}/.npm"
> >  vols="${vols} --volume=${cockroach_toplevel}:/go/src/github.com/cockroachdb/cockroach"
> > +# Add ssh key for terraform tests.
> > +vols="${vols} --volume=${HOME}/.ssh/google_compute_engine.pub:${container_home}/.ssh/google_compute_engine.pub"
>
> The SSH key is absolutely needed in my experience.
> … <#m_4455105680950722193_>
> On Tue, Sep 13, 2016 at 2:06 PM Tamir Duberstein ***@***.***> wrote: In
> build/builder.sh <#9328 (comment)
> <https://github.com/cockroachdb/cockroach/pull/9328#discussion_r78613109>>:
> > @@ -119,6 +119,9 @@ vols="${vols} --volume=${gopath0}/bin/docker_amd64:/go/bin"
> > vols="${vols} --volume=${HOME}/.jspm:${container_home}/.jspm" >
> vols="${vols} --volume=${HOME}/.npm:${container_home}/.npm" >
> vols="${vols} --volume=${cockroach_toplevel}:/go/src/github.com/
> cockroachdb/cockroach" > +# Add ssh key for terraform tests. >
> +vols="${vols} --volume=${HOME}/.ssh/google_compute_engine.pub:${
> container_home}/.ssh/google_compute_engine.pub" I'm pretty sure this is
> not needed or used. On my machine, this file doesn't exist, and I've run
> the terraform tests. — You are receiving this because you are subscribed to
> this thread. Reply to this email directly, view it on GitHub <
> https://github.com/cockroachdb/cockroach/pull/9328/files/
> 9c808420a4223009a7c19924a1db6b4a5bfed5c9#r78613109>, or mute the thread <
> https://github.com/notifications/unsubscribe-auth/
> ABffpiKQ8fHvk7FzD5Nvr4FDpVv9LTzBks5qpuY4gaJpZM4J79Q7> .
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/9328/files/9c808420a4223009a7c19924a1db6b4a5bfed5c9#r78616263>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABdsPJ6V17FVpD_6ov4nKWMqWbzHo0Wmks5qpupfgaJpZM4J79Q7>
> .
>